src/uct/ib/rc/base/rc_iface.c

```diff
@@ -76,6 +76,10 @@ ucs_config_field_t uct_rc_iface_config_table[] = {
    "is a minimum between this value and the maximum value supported by the HW. \n"
    "-1 means no limit.",
    ucs_offsetof(uct_rc_iface_config_t, tm.list_size), UCS_CONFIG_TYPE_UINT},
+
+  {"TM_MAX_BCOPY", "48k",
+   "Maximal size of copy-out sends with eager tag offload protocols.\n",
```

Comment:
"Maximal size of copy-out sends when tag-matching offload is enabled." (no \n)
 - because it affects AMs as well